apps/crn-frontend/src/network/teams/api.ts

```diff
@@ -87,7 +87,7 @@ export const createResearchOutput = async (
   if (!resp.ok) {
     throw new BackendError(
       `Failed to create research output for ${
-        researchOutput.publishingEntity
+        researchOutput.workingGroups.length ? 'Working Group' : 'Team'
```

Comment:
Shouldn't we use that validation function again?

Reply:
The issue here it's that this is a different type, this is the post request

Reply:
Can't it support both types?